cla_backend/apps/checker/admin.py

```diff
@@ -19,7 +20,8 @@ class CallbackTimeSlotAdmin(admin.ModelAdmin):
     change_list_template = "admin/checker/callback-time-slots/custom_change_list.html"
     list_display = ("date", "time", "capacity", "remaining_capacity")
     date_hierarchy = "date"
-    ordering = ("date", "time")
+    ordering = ("-date", "time")
```

Comment:
As we are now only showing the next 7 days, by default, the date should be in ascending order.

This line should read ‘ordering = (“date”, “time”)’

Many thanks!